snet-ui/Flasemessage/index.stories.tsx

```diff
@@ -0,0 +1,10 @@
+import { Meta } from "@storybook/react";
+import FalsemessageComponent from "./";
+
+export default {
```

Comment:
There is a typo in the folder name

Reply:
![image](https://user-images.githubusercontent.com/87366584/140041411-a1eec0b2-be83-4548-aa3a-9775209d90ed.png)
faq

Reply:
![image](https://user-images.githubusercontent.com/87366584/140041490-e883283a-3aec-43bd-9a18-3cd53574477c.png)
false message